src/views/SpaceDelegate.vue

```diff
@@ -286,6 +287,19 @@ onBeforeRouteLeave(async () => {
     </div>
     <Teleport to="#modal">
       <SpaceDelegatesDelegateModal
+        v-if="space.delegationPortal != null"
+        :open="showDelegateModal"
+        :space="space"
+        :address="showUndelegate ? web3Account : address"
+        @close="showDelegateModal = false"
+        @reload="handleReload"
+      />
+      <SpaceDelegatesSplitDelegationModal
+        v-if="
+          space.strategies?.some(
+            ({ name }) => name === DelegationTypes.SPLIT_DELEGATION
+          )
+        "
```

Comment:
This additional modal will never be triggered, since this `/delegate` page is shown when split-delegation is not in used (see https://github.com/snapshot-labs/snapshot/pull/4492/files#diff-5613f74c62003286954dae6762fb9b85d8e077c68ce15e0bd18b715231d84e38R40)

Reply:
removed in [18e49c7](https://github.com/snapshot-labs/snapshot/pull/4492/commits/18e49c74c87feea8d7e4030441dcea6ff532ad34)